### DISCUSSION TITLE
perspective camera with transform rotation as opposed to eye and center

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#227] [Feature Requests] [answered]
I don't know whether it's just me but I always find moving a camera with 2 transforms (eye, centre) really hard to work with.

It would be great to have a perspective camera that's guided with a transform for its position and a rotation for the direction it's pointing.

**Comment by tastez** [comment, marked as answer]
Check out[ TransformView OP](https://cables.gl/op/Ops.Gl.Matrix.TransformView) and [Perspective OP ](https://cables.gl/op/Ops.Gl.Perspective)(for fov adjustment)  probably what you are looking fow 😀

**Reply by marioepsley** [reply]
I never thought to do that but yes thats exactly it. Thank you

**Reply by marioepsley** [reply]
But are we transforming the scene or just the camera?

**Reply by tastez** [reply]
transform view transforms the view not the scene.

**Reply by marioepsley** [reply]
nice one, thank you.
